influx/templates/dashboards/remote_call.json

```diff
@@ -272,7 +272,7 @@
           "position": "overlaid",
           "queries": [
             {
-              "query": "from(bucket: \"windtunnel\")\n  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)\n  |> filter(fn: (r) => r[\"_measurement\"] == \"wt.instruments.operation_duration\")\n  |> filter(fn: (r) => r[\"scenario_name\"] == \"remote_call_rate\")\n  |> filter(fn: (r) => r[\"run_id\"] == v.RunId)\n  |> filter(fn: (r) => r[\"operation_id\"] == \"trycp_app_call_zome\")\n  |> filter(fn: (r) => r[\"zome_name\"] == \"remote_call\")\n  |> filter(fn: (r) => r[\"fn_name\"] == \"call_echo_timestamp\")\n  |> aggregateWindow(every: v.windowPeriod, fn: mean, createEmpty: false)\n  |> yield(name: \"mean\")"
+              "query": "from(bucket: \"windtunnel\")\n  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)\n  |> filter(fn: (r) => r[\"_measurement\"] == \"wt.instruments.operation_duration\")\n  |> filter(fn: (r) => r[\"scenario_name\"] == \"remote_call_rate\")\n  |> filter(fn: (r) => r[\"run_id\"] == v.RunId)\n  |> aggregateWindow(every: v.windowPeriod, fn: mean, createEmpty: false)\n  |> yield(name: \"mean\")"
```

Comment:
All these params in the operation_duration are gone. I'm not sure the data we're referring to, `Remote call round trip (zome call)` still makes sense, though; maybe not, and it should be removed entirely.

Reply:
The zome_name and fn_name should still be available on the automated zome call reporting https://github.com/holochain/wind-tunnel/blob/main/bindings/client/src/app_websocket.rs#L112

The op duration and id are supposed to be coming from here https://github.com/holochain/wind-tunnel/blob/32a2a0857003d2cfd983a3b79479e6db88bb4c39/framework/instruments/src/report/influx_reporter_base.rs#L68

It looks to me like that's still wired up through the derive macro https://github.com/holochain/wind-tunnel/blob/main/framework/instruments_derive/src/lib.rs#L119